tests/in/glsl/math-functions.vert

```diff
@@ -0,0 +1,55 @@
+#version 450
```

Comment:
any reason this doesn't have a corresponding WGSL output snapshot?

Reply:
generation of wgsl output is right now disabled in snapshots, though i'd like to enable it. I can do that in this PR if that's ok.

Reply:
fine by me :)